Index: a
===================================================================
--- a	(revision 1)
+++ a	(revision 2)
@@ -0,0 +1,1 @@
+a
Index: public/images/reorder.gif
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream

Property changes on: public/images/reorder.gif
___________________________________________________________________
Name: svn:executable
   + *
Name: svn:mime-type
   + application/octet-stream

Index: b
===================================================================
--- b	(revision 1)
+++ b	(revision 2)
@@ -0,0 +1,1 @@
+b
